### PR TITLE
#PT.ID API major refactor - Develop

### PR DESCRIPTION
**Pull request by Sylvance**
#PT.ID API major refactor - Develop

##What does this PR do?

This makes changes to the API stopped using basic flask and uses flask-restful as the API. The Models have been changed. Still authentication is the next thing to be done. 

##Description of Task to be completed?

Make the final Pull request that has an update of all the classes in working condition.

##How should this be manually tested?
The Readme file has the installation processes.
What are the relevant pivotal tracker stories?

#PT.ID

Screenshots (if appropriate)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -101,3 +101,7 @@ ENV/
 
 # mypy
 .mypy_cache/
+
+instance
+tree.txt
+migrations
```
Comment (jonathankamau) [inline .gitignore]: a new line is required here

File: .gitignore [hunk 2]
```diff
@@ -101,3 +101,7 @@ ENV/
```
Comment (jonathankamau) [inline .gitignore]: why this space?

Comment (Sylvance) [reply .gitignore]: Yes. Will do so.

File: .gitignore (hunk 1 above)
Comment (Sylvance) [reply .gitignore]: Yes. Will do so.

File: .gitignore (hunk 2 above)
Comment (Sylvance) [reply .gitignore]: Noted.

File: Procfile [hunk 3]
```diff
@@ -0,0 +1 @@
+web: flask run
```
Comment (jonathankamau) [inline Procfile]: New line needed here.

File: app/resources/helper.py [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+    for user_item in user_items:
+        items.append(user_item.json())
+    json = jsonify({'status': 'success',
+        'previous': previous,
+        'next': nex,
+        'count': count,
+        'items': items
+    })
+    json.status_code = 200
+    return json
```
Comment (jonathankamau) [inline app/resources/helper.py]: New line needed here as well.

**Pull request merged**